core/src/main/java/org/infinispan/topology/ClusterTopologyManagerImpl.java

```diff
@@ -179,17 +179,29 @@ public void start() {
    @Stop(priority = 100)
    public void stop() {
       // Stop blocking cache topology commands.
-      updateLock.lock();
+      acquireUpdateLock();
       try {
          clusterManagerStatus = ClusterManagerStatus.STOPPING;
          joinViewFuture.stop();
       } finally {
-         updateLock.unlock();
+         releaseUpdateLock();
       }
 
       cacheManagerNotifier.removeListener(viewListener);
    }
 
+   // This method is here to augment with blockhound as we allow it to block, but don't want the calls
+   // inside the lock to block - Do not move or rename without updating the reference
+   private void acquireUpdateLock() {
+      updateLock.lock();
+   }
+
+   // This method is here to augment with blockhound as we allow it to block, but don't want the calls
+   // inside the lock to block - Do not move or rename without updating the reference
```

Comment:
I would assume the unlock method can't block, so it's just for symmetry

Reply:
Sure, let me try removing actually. I don't quite remember ;)

Reply:
I wanted to keep the method so acquire/release is symmetric, I just wanted to remove the comment and the exception in the blockhound integration.